artichoke-backend/src/extn/core/enumerable/enumerable.rb

```diff
@@ -212,12 +212,14 @@ def flat_map(&block)
   def group_by(&block)
     return to_enum :group_by unless block
 
-    h = {}
+    groups = {}
     each do |val|
-      key = block.call(val)
-      h.key?(key) ? (h[key] << val) : (h[key] = [val])
+      group = block.call(val)
+      values = h.fetch(group, [])
```

Comment:
I think ```s/ h\./ groups./```

Reply:
Thank you @isbear. Fix here https://github.com/artichoke/artichoke/pull/189